packages/babel-plugin-transform-typescript/test/fixtures/imports/import-named-type/output.mjs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Shouldn't this have an output?

Reply:
Yes - fixing now

Reply:
I forgot to set the `onlyRemoveTypeImports` option for these tests 🤦‍♂